packages/code-du-travail-ui/package.json

```diff
@@ -11,29 +11,35 @@
     "umd"
   ],
   "scripts": {
-    "build": "nwb build-react-component --no-demo ",
-    "clean": "nwb clean-module && nwb clean-demo",
-    "prepublishOnly": "npm run build && npm run gh-pages",
-    "gh-pages": "npm run docz:build && gh-pages -d ./doc-build",
+    "build": "run-p -l build:*",
```

Comment:
why not simply use tsc directly ?
```
"build": "tsc --module es2015 --outDir esm",
```

Reply:
because I forgot to reintroduce the "old" build system for the js files :sweat_smile: